static/src/javascripts/bootstraps/enhanced/atoms.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+
+import config from 'lib/config';
+import { services } from 'journalism/services';
+
+const bootstrapAtom = (atomMaker: AtomMaker, atomType: AtomType) => {
+    const atomBuilder = atomMaker[atomType].default(services);
+    [
+        ...document.querySelectorAll(`[data-atom-type='${atomType}']`),
+    ].forEach(atomDom => {
+        const atom = atomBuilder(atomDom).runTry();
+        if (typeof atom === 'string') {
+            // eslint-disable-next-line no-console
+            console.log(
+                `Failed to initialise atom [${atomType}/${atomDom.getAttribute(
+                    'data-atom-id'
+                ) || ''}]: ${atom}`
+            );
+        } else {
+            atom.start();
```

Comment:
this will add a fair chunk of work to do at page load, is that right? should we use `requestIdleCallback` to start each one?

Reply:
Yep, that's a good call. I'll do that.

Reply:
there's no polyfill for that, are you fine with a small abstraction?

Reply:
no it's hard to polyfill – maybe where supported? otherwise unwrapped?

Reply:
what I was going to do 👍